app/views/course/forum/forums/show.html.slim

```diff
@@ -1,11 +1,9 @@
 = page_header format_inline_text(@forum.name) do
-
-= render 'controls'
+  = render 'controls'
 
 table.table.forum-list.table-hover
   thead
     tr
-      th
```

Comment:
hmm... oh wait. It should be 6. The first th should be some icons like unread or something else.